prospectiverowers.html

```diff
@@ -59,60 +59,16 @@
-        <div class="form-group">
-            <label for="rowerExperienceInput">Do you have prior rowing experience? If so, how much?</label>
-            <input type="text" name="Rowing Experience" class="form-control" id="rowerExperienceInput" aria-describedby="rowerExperience" placeholder="Describe your experience">
-            <small id="rowerExperience" class="form-text text-muted">If you don't have any rowing experience, please feel free to describe any athletic experience!</small>
-        </div>
-        <div class="form-group">
-            <label for="rowerCoxswainInput">Are you intereseted in becoming a coxwain?</label>
-            <input type="text" name="Coxswain-Interest" class="form-control" id="rowerCoxwainInput" aria-describedby="rowerCoxwain" placeholder="Please enter yes if interested, no if not interested">
-            <small id="rowerCoxwain" class="form-text text-muted">Coxswains are the leaders within the boat. Our boats consist of 4 
-                    rowers and 1 coxswain. The coxswain is in charge of the whole boat. 
-                    They decide which way to steer and command the rowers. This position 
-                    requires strong leadership skills and the ability to give directions to 
-                    large groups of people. Coxswains are typically smaller in stature in 
-                    order to fit in the coxswain's seat within the boat.</small>
-        </div>
-        <div class="form-group">
-            <label for="rower2KInput">If you have rowed before what was your best 2000m score?</label>
-            <input type="text" name="2K" class="form-control" id="rower2KInput" aria-describedby="rower2K" placeholder="Enter your 2K score">
-            <!--<small id="rower2K" class="form-text text-muted">We'll never share your email with anyone else.</small>-->
-        </div>
-        <div class="form-group">
-            <label for="rowerPreferredContactInput">What is the best way to get in touch with you?</label>
-            <input type="text" name="GetInTouch" class="form-control" id="rowerPreferredContactInput" aria-describedby="rowerPreferredContact" placeholder="Enter your preferred contact method">
-            <!--<small id="rower2K" class="form-text text-muted">We'll never share your email with anyone else.</small>-->
-        </div>
-        <div class="form-group">
-            <label for="rowerContactMeInput">Would you like to be added to the email contact list and recieve crew updates?</label>
-            <input type="text" name="Would You like to be added to our mailing?" class="form-control" id="rowerContactMeInput" aria-describedby="rowerContactMe" placeholder="Yes/No">
-            <!--<small id="rower2K" class="form-text text-muted">We'll never share your email with anyone else.</small>-->
-        </div>
-        <button type="submit" class="btn btn-primary">Submit</button>
-    </form>
+    <!-- <h2 class="form-title-local">Prospective Rower Interest Form</h2> --> 

```

Comment:
If we no longer need this, we can delete it.

Reply:
Gotcha, I'll try to resolve both issues after class. To update it, do I have to go back to my code and then make a new pull request, or...?

Reply:
So after you make your changes, you can do the following commands

`git add .` to add all changed files
`git commit -m "Message here"` This bunches up your changes into a 'commit', which is basically just a group of changes
`git push` This pushes the commits up to the 'remote' git repository AKA GitHub Repo

After you do all that, you'll see the changes reflected in this pull request

Reply:
...I think I did it. Did I?